src/lib.rs

```diff
@@ -50,6 +61,8 @@ extern crate unicode_width;
 extern crate vte;
 extern crate xdg;
 extern crate base64;
+extern crate winit;
+extern crate image;
```

Comment:
Can this be conditionally imported as well? It looks like it's only used on Windows.